### ISSUE TITLE
suggestion: add a generator from JSON test data to Junit test cases.

### ISSUE THREAD

**Issue by monperrus** [jkoppel/QuixBugs#2]
Follow-up of #1 

For running Java repair tools, one usually needs Junit test cases. It seems possible to generate those test cases based on the JSON test data. Both the generator and the generated Java/Junit files could be added to the repo.

**Comment by jkoppel** [comment]
Noted. Can you provide an example output of the kind that would help you?

**Comment by monperrus** [comment]
OK.

**Comment by monperrus** [comment]
For the first line of https://github.com/jkoppel/QuixBugs/blob/master/json_testcases/bitcount.json

The generated test method would be
```java
@Test
void test_bitcount_line1() {
  assertEquals(7, bitcount(127));
}
```

**Comment by SophieHYe** [comment]
Hi Martin,
The Junit generator is done, could you please review if it is your expected one. #4

**Comment by monperrus** [comment]
Super cool to see progress on this, thanks a lot @SophieHYe .

I would suggest to push the generated Junit test cases as well, so that users get them right after cloning.

May be in a separate pull request?

**Comment by monperrus** [comment]
Closed per #4, thanks a lot @SophieHYe
